orchagent/portsorch.cpp

```diff
@@ -3431,6 +3462,14 @@ bool PortsOrch::removeBridgePort(Port &port)
         return false;
     }
 
+    //Flush the FDB entires corresponding to the port
+    if (!flushFDBEntries(port.m_bridge_port_id))
```

Comment:
one port can be added to multi VLAN, when we flush the mac we should only flush the mac address learned in VLAN which is moving out? what do you think?

Reply:
Let us say, port Ethernet0 is a member of VLAN 2. The FDB entry that is learnt for the port is in VLAN 1. But, once we try to remove Ethernet 0 from being member of VLAN2, looks like we have to remove FDB entries of the bridge port corresponding to VLAN 2 as well as VLAN 1. Is that right?

Reply:
@kebo, please ignore my above comment. removeBridgePort() member function is called only if the port is no more part of any VLANs. So, the call to flush FDB entries with no VLAN information is right. It will flush all the Fdb etries corresponding to the port.